docs/tools/dotnet-Commands.md

```diff
@@ -9,7 +9,7 @@ ms.topic: conceptual
 
 # dotnet commands
 
-The `dotnet` command-line interface, which runs on Windows, Mac OS X, and Linux, provides a number of essential nuget.exe commands as listed below. If dotnet satisfies your needs, it's not necessary to use `nuget.exe`.
+The `dotnet` command-line interface provides several NuGet commands and supports Windows, Linux, and macOS.
```

Comment:
The `Windows, Linux, and macOS` styling follows that of [the dotnet tool's homepage](https://dotnet.microsoft.com/)